pkg/datasources/common.go

```diff
@@ -114,6 +145,30 @@ func handleLimitFrom(d *schema.ResourceData, setField **sdk.LimitFrom) {
 	}
 }
 
+func handleIn(d *schema.ResourceData, setField **sdk.In) error {
+	if v, ok := d.GetOk("in"); ok {
+		in := v.([]any)[0].(map[string]any)
+		if v, ok := in["account"]; ok && v.(bool) {
```

Comment:
nit: only one of these fields should be set but this is on the schema level, so maybe switch would be better here (or if-elses chain)?